src/Boids.hs

```diff
@@ -22,7 +22,7 @@ data World = World { worldBoids :: [Boid]
 data Boid = Boid { boidPosition :: Point
                  , boidHeading :: Float
                  , boidSteer :: Float
-                 } deriving Show
+                 } deriving (Show, Eq, Ord)
```

Comment:
I'm not considering this comment blocking. Boids will derive from Eq and Ord until I resolve #27.